### PR TITLE
Bug fix for the "build html5app" commands use of URLs

### PR DESCRIPTION
**Pull request by ricallinson**
The build html5app command uses the URLs from the resource store to generate files. The URLs that it gets do not have the value of "appConfig.staticHandling.prefix" prepend if it was defined. This patch simply adds the prefix if it was found to the front of the URLs.

### CODE REVIEW

**Comment by drewfish** [comment]
The res.url should definitely have the staticHandler.prefix attached.  If not, that's a bug elsewhere, and not a bug in commands/build.

**Comment by drewfish** [comment]
Look in `app/addons/rs/url.server.js`, that's the _only_ place that each resource's URL is set.  It's possible that there's a bug elsewhere that is preventing addon-rs-url from getting the right appConfig.

**Comment by drewfish** [comment]
Oh, addon-rs-url is just using the _static_ appConfig, so the URLs aren't context-sensitive.  That is definitely a bug (and be the bug you're running into).

File: source/lib/app/commands/build.js [hunk 1]
```diff
@@ -161,6 +161,7 @@ exports.buildhtml5app = function(cmdOptions, store, config, destination,
         contextObj = libqs.parse(cmdOptions.context),
```
Comment (drewfish) [inline source/lib/app/commands/build.js:161]: (BTW, all other users of `contextObj` should probably be using `config.context` instead too.)

**Comment by ricallinson** [comment]
Thanks Drewfish. I'll close this pull and open a new one once I've gone down the rabbit hole.

**Pull request closed without merging**